distribution/lib/Standard/Table/0.0.0-dev/src/Refined_Types/Numeric_Column.enso

```diff
@@ -0,0 +1,31 @@
+## A column that contains numeric values.
+type Numeric_Column
+    private Value column operations_implementation
```

Comment:
- Great to see manual implementation of _type classes_ pattern
- The field name `operations_implementation` is quite long
- Usual name for set of operations is _dictionary_ - e.g. `dict`

Reply:
I don't think the length of this name is causing any problems, I prefer clarity, we have autocomplete in editors.

`dict` is easy to confuse with our `Dictionary` type which is a different thing.

Reply:
- consistency with [other _type classes_ manual implementations](https://github.com/enso-org/enso/blob/2eb1bcd475680eef60c01f10b7f41b4aa989fb04/distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso#L115) is also important
- naming a _type classes dictionary_ `operations_implementation` is too long for my taste
- however it's `private` constructor - e.g. not part of the API
- I don't want to waste my time with discussion on naming _"privates"_